src/explorer/siteNodeBase.ts

```diff
@@ -74,9 +74,9 @@ export class SiteNodeBase extends NodeBase {
 
         if (util.isSiteDeploymentSlot(this.site)) {
             const slotName = util.extractDeploymentSlotName(this.site);
-            await this.webSiteClient.webApps.stopSlot(rgName, siteName, slotName);
+            await this.webSiteClient.webApps.stopSlotWithHttpOperationResponse(rgName, siteName, slotName);
```

Comment:
Discussed offline and I don't think 'stopSlotWithHttpOperationResponse' is necessary (but also wouldn't hurt)